### ISSUE TITLE
Ajout de lien hypertexte sur média image

### ISSUE THREAD

**Issue by jubonhomme** [l-atelier-des-chercheurs/dodoc#388]
Bonjour,

Y a t il un moyen d'insérer un lien hypertexte sur une photo dans une publication page à page ? 
C'est possible pour le texte dans l'outil texte avec le bouton lien, mais je n'ai pas trouvé comment faire pour une image, ni dans les paramètres du média ni dans le CSS de la publication.

Pour l'instant, on peut le faire en ajoutant du texte avec un lien en dessous de l'image, mais peut-on associer le lien directement sur l'image ?

julien

**Comment by louis-ev** [comment]
Non, ça n’est pas possible… mais ça aurait du sens de le permettre oui.
Dans la mesure ou la recette "page à page" vise plutôt à produire des documents imprimés ça n’avait pas trop de sens, mais elle est de plus en plus utilisée pour des pages web.

À voir si la solution ne serait pas une autre recette dédiée à la création de page web scrollable ? On avait expérimenté ça dans do•doc 6, c’était prometteur. Si l’occasion se présente de développer une recette "site web" ça aurait toute sa place je pense !

**Comment by louis-ev** [comment]
Je propose de repousser à la v10, ça ne me paraît pas indispensable à la v9.

**Comment by jubonhomme** [comment]
J'ai eu une demande récemment de ce cette possibilité de pouvoir mettre un lien hypertexte sur une image (je pense que cette habitude viens des "Genialy" )
Ça pourrait être une option dans le bandeau de gauche quand un média image est sélectionné

**Comment by louis-ev** [comment]
Tu peux m'en dire plus ? Publication page à page uniquement ? Sur n'importe quel type de média ?

**Comment by jubonhomme** [comment]
La plupart des retours sur ce point correspondent effectivement à des demandes pour faire comme "Genially"
Par exemple ça 
https://view.genial.ly/659833f81dfdad0014f52ccb/presentation-apps-education

Ce serait donc plus la possibilité de faire des boutons avec des liens internes (ancres) dans un document.
Sur une page j'ai plusieurs images et si je clique sur une image ça ouvre une partie de la publication

Mais on s'éloigne d'une publication simple à réaliser où je dépose des textes / des médias et je fais une mise en page très rapidement.
J'ai peur qu'en ajoutant trop de possibilités on perde les utilisateurs dans des menus trop complexes.

Peut-être qu'on pourrait imaginer ce type de possible dans une publication markdown pour utilisateurs avancés.
Voir par exemple ce que propose Cédric Eyssette 
https://codimd.apps.education.fr/5vGt6F4zRjOyf7wAwfC1og#

**Comment by jubonhomme** [comment]
Non prioritaire -> V10.1 ?

**Comment by louis-ev** [comment]
J'aime bien l'idée de liens vers une autre page, pour permettre la création d'une publi dont vous êtes le héro par exemple :)
Mais oui 10.1 c'est bien !

**Comment by louis-ev** [comment]
Possible sur la v12, un lien veers une page web (qui s'ouvre dans un nouvel onglet) ou vers une autre page d'une même document page à page.

<img width="1235" height="792" alt="Image" src="https://github.com/user-attachments/assets/8d55eba0-6725-418a-9b9f-4738a614d11f" />
